src/UseCases/Embed/EmbedRequestBuilder.php

```diff
@@ -15,12 +17,17 @@ public static function argumentsToRequest( array $arguments, bool $showEditButto
 		$language = $normalizedArguments['lang'] ?? null;
 		$line = $normalizedArguments['line'] ?? null;
 		$specificLines = $normalizedArguments['specificLines'] ?? null;
+		$render = $normalizedArguments['render'] ?? null;
+
+		/** @var bool */
+		$markdownByDefault = MediaWikiServices::getInstance()->getMainConfig()->get( 'ExternalContentRenderMarkdownByDefault' );
```

Comment:
Ideally we should also have a test to make sure the new config value is used when creating the request.

Reply:
You made me realize that this code is accessing global config. The config should be injected. The only place where you are allowed to talk to `MediaWikiServices::getInstance()` is `EmbedExtensionFactory` and entry points.